pkg/storage/storage.go

```diff
@@ -668,3 +679,71 @@ func (s *storageModule) Monitor(ctx context.Context) <-chan pkg.PoolsStats {
 
 	return ch
 }
+
+func (s *storageModule) periodicallyCheckDiskShutdown() {
+	ticker := time.NewTicker(5 * time.Minute)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				log.Info().Msg("Checking pools for disks that should be shutdown...")
+				s.shutdownDisks()
+			case <-quit:
+				ticker.Stop()
+				return
+			}
+		}
+	}()
+}
+
+// shutdownDisks will check the disks power status.
+// If a disk is on and it is not mounted then it is not supposed to be on, turn it off
+func (s *storageModule) shutdownDisks() {
+	for _, pool := range s.pools {
+		for _, device := range pool.Devices() {
+			log.Debug().Msgf("checking device: %s", device.Path)
+			on, err := checkDiskPowerStatus(device.Path)
+			if err != nil {
+				if err, ok := err.(*exec.ExitError); ok {
+					log.Err(err).Msgf("skipping device")
+					// if cmd exits with exit error the device is shutdown
+					continue
+				}
+			}
+
+			_, mounted := pool.Mounted()
+			if mounted {
+				continue
+			}
+
+			if on {
```

Comment:
Then you don't need to do this check anymore since you won't be here if disk is off